docs/dns-health-checks.md

```diff
@@ -0,0 +1,86 @@
+# DNS Health Checks
+DNS Health Checks are a crucial tool for ensuring the availability and reliability of your multi-cluster applications. Kuadrant offers a powerful feature known as DNSPolicy, which allows you to configure and verify health checks for DNS endpoints. This guide provides a comprehensive overview of how to set up, utilize, and understand DNS health checks.
+
+## What are DNS Health Checks?
+DNS Health Checks are a way to assess the availability and health of DNS endpoints associated with your applications. These checks involve sending periodic requests to the specified endpoints to determine their responsiveness and health status. by configuring these checks via the [DNSPolicy](https://github.com/Kuadrant/multicluster-gateway-controller/blob/main/docs/dns-policy.md), you can ensure that your applications are correctly registered, operational, and serving traffic as expected.
```

Comment:
```suggestion
DNS Health Checks are a way to assess the availability and health of DNS endpoints associated with your applications. These checks involve sending periodic requests to the specified endpoints to determine their responsiveness and health status. by configuring these checks via the [DNSPolicy](./dns-policy.md), you can ensure that your applications are correctly registered, operational, and serving traffic as expected.
```

I'd use a relative link here so when this is in the docs website it stays in that website instead of redirecting to Github